recourse_methods/test_mrmc_method.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+from unittest import mock
+import numpy as np
+import pandas as pd
+from recourse_methods import mrmc_method
+
+
+SMALL_CUTOFF = 0.2
+LARGE_CUTOFF = 2
+DEGREE = 2
+volcano_alpha_large_cutoff = mrmc_method.get_volcano_alpha(
```

Comment:
What was the motivation to set these as global variables/shortcuts? I would recommend using these directly (`mrmc_method.get_volcano_alpha()`) in the tests themselves.